### DISCUSSION TITLE
Not recognizing css file

### DISCUSSION THREAD

**Discussion by n-ce** [github-linguist/linguist#5537] [Classification] [answered]
Here's the repo https://github.com/n-ce/Formify

**Comment by lildude** [comment, marked as answer]
It's because of [this line](https://github.com/n-ce/Formify/blob/f6f3ef34bff39cd9f0a3e486b9dd19a6804b2c27/style.css#L12)… it's so disproportionately long compared to the number of lines in the file that it's tripping the minification check at https://github.com/github/linguist/blob/32ec19c013a7f81ffaeead25e6e8f9668c7ed574/lib/linguist/generated.rb#L137-L151
